spec/proto/basic.spec.coffee

```diff
@@ -27,11 +27,11 @@ describe "Basic setup", ->
     expect($F(select)).toBe ""
 
     container = div.down(".chosen-container")
-    container.simulate("mousedown") # open the drop
+    Chosen.trigger(container, "mousedown") # open the drop
```

Comment:
Well, as Chosen does not need a full-fledged event triggering system (as it does not need to trigger mouse events), the simpler would probably be to keep using an external library for that in the testsuite